scripts/test/get_all_tests.sh

```diff
@@ -2,5 +2,6 @@
 
 SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
+# shellcheck disable=SC2207,SC2086
```

Comment:
should we add a comment, why we disable here?

Reply:
Done